ruby/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM clearlinux:latest
+MAINTAINER hongzhan.chen@intel.com
+
+RUN swupd update $swupd_args && \
```

Comment:
Please add "--no-boot-update" as parameter to "swupd update". 
The update on container doesn't need update the boot partition.